### PR TITLE
MBS-1 Ensuring the Backup Server Models Meet Requirements

### PR DESCRIPTION
**Pull request by samuelowino**
* Pending Unit Tests - my bad *

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,6 +14,25 @@
...
+		<developer>
+			<name>Samuel Owino</name>
+			<email>samuelowino43@gmail.com</email>
+			<organization>mUzima</organization>
+			<organizationUrl>www.muzima.org</organizationUrl>
+		</developer>
+
+		<developer>
+			<name>Mhawila Mhawila</name>
+			<email>nill</email>
```
Comment (mhawila) [inline pom.xml]: Name: Mhawila Mhawila (Willa) - You don't want people to be confused. In fact I simply use Willa Mhawila
email: a.mhawila@gmail.com
organization email: www.vumc.org

File: src/main/java/org/muzima/turkana/service/impl/UsageLogsServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    @Override
+    public void saveAllLogs(List<UsageLogs> usageLogsList) {
+        usageLogsRepository.saveAll(usageLogsList);
+    }
+
+    @Override
+    public UsageLogs getUsageLogsEntry(String uuid) {
+        return usageLogsRepository.findById(uuid).orElseGet(UsageLogs::new);
```
Comment (mhawila) [inline src/main/java/org/muzima/turkana/service/impl/UsageLogsServiceImpl.java]: The Spring data standard implementation is going to compare against the property named "id" which is not what you want according to this call.

**Review by mhawila — COMMENTED** [review]
Like I said we need to talk.

File: src/main/java/org/muzima/turkana/service/impl/UsageLogsServiceImpl.java (hunk 2 above)
Comment (samuelowino) [reply src/main/java/org/muzima/turkana/service/impl/UsageLogsServiceImpl.java]: Resolved: This now uses Long: id instead of the previously defined {uuid}

**Pull request merged**